app/models/student.rb

```diff
@@ -280,4 +280,18 @@ def primary_payment_method_belongs_to_student
       errors.add(:primary_payment_method, 'must belong to you.')
     end
   end
+
+  def self.pull_info_from_crm(email)
+    close_io_client = Closeio::Client.new(ENV['CLOSE_IO_API_KEY'], false)
+    lead = close_io_client.list_leads('email:' + email)
+    if lead.total_results == 1
+      name = lead.data.first.contacts.first.name
+      course = Course.find_by(description: lead.data.first.custom.Class)
+      if name && course
+        return {name: name, course_id: course.id}
```

Comment:
`return` statements are usually a bad idea in Ruby. Here, it will be easier to read your code if you complete the if/else/end block, rather than using `return` statements.